sqlmesh/core/renderer.py

```diff
@@ -61,11 +59,22 @@ def __init__(
         self._default_catalog = default_catalog
         self._quote_identifiers = quote_identifiers
         self.update_schema({} if schema is None else schema)
-        self._cache: t.Dict[CacheKey, t.List[t.Optional[exp.Expression]]] = {}
+        self._cache: t.List[t.Optional[exp.Expression]] = []
+        self._model_fqn = model_fqn
```

Comment:
I don't see this being passed into the `ExpressionRenderer` anywhere? Shouldn't we do this while rendering pre-/post- statements?

Reply:
done